package.json

```diff
@@ -16,7 +16,7 @@
   "dependencies": {
     "@duckduckgo/autoconsent": "^5.1.0",
     "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#8.2.0",
-    "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#4.36.0",
+    "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#09-14-WIP_adding_android_messaging_support",
```

Comment:
This will point to the right version before merging

Reply:
What is the right version?